<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>data/pyPlayer.png</normaloff>data/pyPlayer.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.869, fx:0.5, fy:0.5, stop:0 rgba(99, 168, 218, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label_record">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Record:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
